PrebidMobile/PrebidMobile-core/proguard-rules.pro

```diff
@@ -1,12 +1,12 @@
 
 # GAM
--keep class com.google.android.gms.ads.doubleclick.PublisherAdRequest {
+-keep class com.google.android.gms.ads.doubleclick.AdManagerAdRequest {
```

Comment:
I think we should add new rules for new classes (instead of changing the old ones). The reason for this is that Prebid SDK implies support of 2 versions (>= 19.7.0 (including 20.0.0)) at once and this rules might break support for publishers on version 19.7.0. 
Also you forgot to change the package name (they also changed for new classes) so these rules won't work for either of versions.

Reply:
done